middleware/sync.js

```diff
@@ -23,11 +23,23 @@ import { activeRouteQueries$ } from '../util/routeUtils';
 import { fetchQueries } from '../util/fetchUtils';
 
 /**
```

Comment:
It's not set up to actually generate docs right now, and I'm ambivalent about adding that in, but I can open a task for it if we decide it would be valuable. I like the amount of "let's add some code comments about this" comments coming through PRs at the moment.